walkthroughs/week-1-setup/intro-walkthrough.md

```diff
@@ -94,7 +94,7 @@ Wherever you see a file that ends with `-walkthrough.md`, you can run the
 this command runs the walkthrough you're currently reading:
 
 ```bash
-teachme ~/step/walkthroughs/week-1-setup/intro-walkthrough.md
+
```

Comment:
Did you mean to remove just this single line?

Reply:
> Did you mean to remove just this single line?

This was an error. I corrected it in #6